docs/source/configuration/overview.mdx

```diff
@@ -540,7 +540,9 @@ By default, the Apollo Router stores the following data in its in-memory cache t
 
 You can configure certain caching behaviors for generated query plans and APQ (but not introspection responses). For details, see [In-memory caching in the Apollo Router](./in-memory-caching/).
 
-**If you have a GraphOS Enterprise plan,** you can also configure a Redis-backed _distributed_ cache that enables multiple router instances to share cached values. For details, see [Distributed caching in the Apollo Router](./distributed-caching/).
+**If you have a GraphOS Enterprise plan:** 
+- You can configure a Redis-backed _distributed_ cache that enables multiple router instances to share cached values. For details, see [Distributed caching in the Apollo Router](./distributed-caching/).
```

Comment:
@shorgi , do we want to start moving away from italics except for new terminology _now_ or do it all at once later?

I've personally been cutting back on usage, but figure we can do the switch over to new terms using italics all at once.